src/constants/collection/index.ts

```diff
@@ -56,6 +62,7 @@ export const UPLOAD_NOT_ALLOWED_COLLECTION_TYPES = new Set([
     CollectionSummaryType.outgoingShare,
     CollectionSummaryType.sharedOnlyViaLink,
     CollectionSummaryType.trash,
+    CollectionSummaryType.uncategorized,
```

Comment:
uploading to uncategorized is allowed, as it is an actual collection

Reply:
also, pls merge the main once, have removed `outgoingShare` and `sharedOnlyViaLink` , as `outgoingShare` and `sharedOnlyViaLink` are  user owned collections, which user should be able to upload to.

only incomingShare should be in this list as, they are someone else's collection in which user can't upload